### PR TITLE
reverse cull face for models with odd nb of negative scales

### PR DESCRIPTION
**Pull request by tkazik**
Fix for https://github.com/AnalyticalGraphicsInc/cesium/issues/6304

### CODE REVIEW

**Comment by pjcozzi** [comment]
Thanks again @tkazik, we received your CLA.

Anyone know why @cesium-concierge didn't comment on this PR?

File: Source/Scene/Model.js [hunk 1]
```diff
@@ -2951,6 +2953,16 @@ define([
 
             var rs = rendererRenderStates[material.technique];
 
+            // reverse for negative scales
+            var nodeScales = gltfNode.scale;
+            var scalesProd = nodeScales[0] * nodeScales[1] * nodeScales[2];
```
Comment (lilleyse) [inline Source/Scene/Model.js]: Ideally I think we'd want to use the `computedMatrix` in some way, but I'm still not sure how.

Some of the problems here might be if the gltf node's scale is undefined (if it's using a matrix), or if the node's parent is really the one with the negative scale.

Comment (tkazik) [reply Source/Scene/Model.js]: We could check if a node has TRS values or a matrix and then either take the product of scales for the TRS or the determinant for the matrix.
However, you are right about the parenting issue...checking the node itself is not sufficient.

**Comment by lilleyse** [comment]
No update yet.

The fix here will work in some cases but may not be robust enough. Unfortunately a full fix is surprisingly involved since our loader generates render states before updating the node hierarchy (we would need the reverse).

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @tkazik! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by lilleyse** [comment]
@tkazik I'm going to close this PR for now. But anyone can carry on the work here, with the warning that a complete fix may require a bit of work.

**Pull request closed without merging**